.circleci/config.yml

```diff
@@ -118,7 +118,9 @@ jobs:
             .circleci/copy-configs
       - run:
           name: Run integration test
-          command: .circleci/retry.sh 3 ./gradlew cAT --no-daemon
+          command: |
+              .circleci/retry.sh 3 ./gradlew connectedDebugAndroidTest jacocoFullReport --no-daemon
+              bash <(curl -s https://codecov.io/bash)
```

Comment:
I'm always a little iffy on these pipe-it-to-bash incantations. Can we vendor this script locally? If not, can we check for a checksum to ensure we're downloading what we expect?

Reply:
Updated to check a hash